### PR TITLE
Fix typo / line wrapping in `team.dart`

### PR DESCRIPTION
**Pull request by creativecreatorormaybenot**
I noticed this while reading through the docs and it really stands out / was not nice to read through, so I made sure to align the line lengths and add the space where needed.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by Hixie** [comment]
test-exempt: only affects comments

File: packages/flutter/lib/src/gestures/team.dart
Comment (Hixie) [inline packages/flutter/lib/src/gestures/team.dart:120]: if you intend this to be a new paragraph in the API docs you'll need a blank line here too, fwiw.

Comment (creativecreatorormaybenot) [reply packages/flutter/lib/src/gestures/team.dart:120]: Thanks - from what I see this was not a new paragraph previously either. Although that does not make sense from a Markdown perspective because the line will just be wrapped after the other.

I will add two trailing spaces for now (which will put the "The captain ..." sentence into a new line without a blank line in between).

**Comment by goderbauer** [comment]
@creativecreatorormaybenot Can you please fix what the analyzer is complaining about?

**Comment by creativecreatorormaybenot** [comment]
@goderbauer done. It was on purpose actually (double trailing space, which creates a new line in Markdown), but I suppose the analyzer does not support it..

**Comment by Piinks** [comment]
(triage) Hey @creativecreatorormaybenot can you update your branch with the latest from master? It looks like the failing CI checks here are unrelated. Thanks!

**Comment by creativecreatorormaybenot** [comment]
@Piinks done.

**Review by Piinks — APPROVED** [review]
LGTM!

**Comment by Piinks** [comment]
I think this needs a second review per the new policy that landed in https://github.com/flutter/cocoon/pull/1434

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**
